presto-verifier/src/main/java/com/facebook/presto/verifier/checksum/ChecksumValidator.java

```diff
@@ -55,14 +53,11 @@ public Query generateChecksumQuery(QualifiedName tableName, List<Column> columns
         return simpleQuery(new Select(false, selectItems.build()), new Table(tableName));
     }
 
-    public Map<Column, ColumnMatchResult> getMismatchedColumns(List<Column> columns, ChecksumResult controlChecksum, ChecksumResult testChecksum)
+    public List<ColumnMatchResult<?>> getMismatchedColumns(List<Column> columns, ChecksumResult controlChecksum, ChecksumResult testChecksum)
```

Comment:
Nit: I'd move this to a separate commit, but it's up to you.

Reply:
Done